pkg/restore/restore.go

```diff
@@ -1294,15 +1294,6 @@ func (ctx *restoreContext) restoreItem(obj *unstructured.Unstructured, groupReso
 		}
 	}
 
-	objStatus, statusFieldExists, statusFieldErr := unstructured.NestedFieldCopy(obj.Object, "status")
-	// Clear out non-core metadata fields and status.
-	if obj, err = resetMetadataAndStatus(obj); err != nil {
-		errs.Add(namespace, err)
-		return warnings, errs, itemExists
-	}
-
-	ctx.log.Infof("restore status includes excludes: %+v", ctx.resourceStatusIncludesExcludes)
-
```

Comment:
the concern that @reasonerjt  had in https://github.com/vmware-tanzu/velero/issues/8094#issuecomment-2275133330 still holds. 
Please pitch in others  - CC: @sseago, if this could impact any RIA flows.

Reply:
@anshulahuja98 I'm unaware of any RIA that currently modifies status -- if I'm remembering correctly, don't we currently strip the status field before RIAs are run? I think that's one of the reasons we added unmodified item (`itemFromBackup`) to the RIA input, as it still has the full status available for plugins to reference.